app/src/main/java/org/grammatek/simacorrect/network/ConnectionManager.kt

```diff
@@ -24,6 +25,7 @@ class ConnectionManager {
     private var _threadPoolExecutor: ScheduledThreadPoolExecutor? = null
     private var _periodicFuture: ScheduledFuture<*>? = null
     private val _apiServer: String = "yfirlestur.is"
```

Comment:
needs to be changed to `yfirlestur.grammatek.com`, but I'd propose to make a new MR when our service is available